docs/src/development.md

```diff
@@ -18,7 +18,7 @@ package, which tracks changed files and re-loads them automatically. Therefore,
 it is *highly recommended* to first install Revise with the following command in Julia:
 To enter the package REPL mode, press `]` in the standard Julia REPL mode. Then, execute
 ```julia-repl
-(@v1.6) pkg> add Revise
+(@v1.7) pkg> add Revise
```

Comment:
It's nice that you changed it but I wouldn't consider it necessary for future updates.